llm_bioinformatics_research/src/Components/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,148 @@
+        setMessage(data.message);
+  
+        // Delay before redirecting to login page
+        setTimeout(() => {
+          navigate('/login'); // Redirect to login after 3 seconds
+        }, 3000); // 3 seconds delay
+      } else {
+        setError(data.message);
+      }
+    } catch (error) {
+      console.error("Error resetting password:", error);
+      setError("Failed to reset password. Please try again.");
+    }
+  };
+
+  return (
+    <Container maxWidth="sm" className="reset-container">
+      <Box className="reset-box">
+        <Typography variant="h4" gutterBottom>
+          Reset Password
+        </Typography>
+        <form onSubmit={handleSubmit} noValidate>
+          <TextField
+            label="New Password"
+            type={passwordVisible ? 'text' : 'password'}
+            variant="outlined"
+            fullWidth
+            margin="normal"
+            value={newPassword}
+            onChange={(e) => setNewPassword(e.target.value)}
+            error={!!passwordError}
+            helperText={passwordError}
+            InputProps={{
+              endAdornment: (
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={() => setPasswordVisible(!passwordVisible)}
+                  edge="end"
+                >
+                  {passwordVisible ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              ),
+            }}
+          />
+          <TextField
+            label="Confirm Password"
+            type={confirmPasswordVisible ? 'text' : 'password'}
+            variant="outlined"
+            fullWidth
+            margin="normal"
+            value={confirmPassword}
+            onChange={(e) => setConfirmPassword(e.target.value)}
+            error={!!confirmPasswordError}
+            helperText={confirmPasswordError}
+            InputProps={{
+              endAdornment: (
+                <IconButton
+                  aria-label="toggle confirm password visibility"
+                  onClick={() => setConfirmPasswordVisible(!confirmPasswordVisible)}
+                  edge="end"
+                >
+                  {confirmPasswordVisible ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              ),
+            }}
+          />
+          {error && (
+            <Typography variant="body2" color="error" className="reset-error">
+              {error}
+            </Typography>
+          )}
+          {message && (
+            <Typography variant="body2" color="success" className="reset-success">
+              {message}
+            </Typography>
+          )}
+          <Button
+            type="submit"
+            variant="contained"
+            color="primary"
+            fullWidth
+            className="reset-button"
+          >
+            Reset Password
+          </Button>
+        </form>
+      </Box>
+    </Container>
+  );
+}
+
```

Comment:
Same thing for this page. I think we should update the layout of this page to be similar to the login/sign up pages to ensure consistency across the application. Integrating the material UI library is necessary for those changes.